mission_3/routes/handler/errorHandler.js

```diff
@@ -0,0 +1,7 @@
+function errorHandler(err, req, res, next) {
+    console.error(`[ERROR] ${err.message}`);
+
+    res.status(err.status || 500).json({ message: "An error has occurred during processing sql" });
+}
+
+export default errorHandler;
```

Comment:
이렇게 별개로 export해주는 것보다 각 메서드/변수/클래스를 선언할 때 바로 export 해주는 것이 나중에 유지보수하기에는 더 좋습니다. 선언부와 export 구문이 근접하면 관심사가 같은 줄에 있어서 편집하기 용이해요.
```suggestion
export function errorHandler(err, req, res, next) {
```